### DISCUSSION TITLE
Failed to delete an entity using attu

### DISCUSSION THREAD

**Discussion by sunwhite5** [milvus-io/milvus#34776] [Q&A and General discussion]
Use attu to query the entity, and then select and delete it. Query the entity again, it can still be queried.
attu version is 2.4.4
milvus server version is 2.4.5

**Comment by congqixia** [comment]
Just test this operation locally. Cannot reproduce:
1. Query data in `Data` tab
2. Select entry to delete
3. Click delete button
4. Type `drop` to confirm
5. Query with id == xxxx, no data returned

Did I miss anything?

**Reply by xiaofan-luan** [reply]
attu entity is not accurate. Don't refer to the entity count
